src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGeneratorTest.java

```diff
@@ -48,6 +50,26 @@
 
     private static FileText fileText;
 
+    private final CloseAndFlushTestByteArrayOutputStream outStream =
+            new CloseAndFlushTestByteArrayOutputStream();
+
+    private final String[][] encodings = {
+            {"<", "&lt;"},
+            {">", "&gt;"},
+            {"'", "&apos;&apos;"},
+            {"\"", "&quot;"},
+            {"&", "&amp;"},
+            {"&lt;", "&amp;lt;"},
+            {"abc;", "abc;"},
+            {"&#0;", "&amp;#0;"},
+            {"&#0", "&amp;#0"},
+            {"&#X0;", "&amp;#X0;"},
+            {"\u0001", "#x1;"},
+            {"\u0080", "#x80;"},
+            {"\n", "&#10;"},
+            {"\r", ""},
+    };
```

Comment:
please change to map and make local variable

Reply:
Ok.

Reply:
Done

Reply:
@AkMo3 it is changed to map, but it is not a local variable but field. This map is not needed anywhere else but your test method, so please move it to your test

Reply:
Done.